<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConffileWidget</class>
 <widget class="QWidget" name="ConffileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="filenameLabel">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QRadioButton" name="keepRadioButton">
     <property name="text">
      <string>Keep old</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="replaceRadioButton">
     <property name="text">
      <string>Replace with new</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showDiffButton">
     <property name="text">
      <string>Show diff</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
